features/mesh/services/Item.feature

```diff
@@ -141,9 +121,6 @@ Feature: mesh / services / item
           offset: 0
           size: 50
         """
-      And the URL "/meshes/default/dataplanes+insights" responds with
-        """
-        """
       And I click the "$button-clear-search" element
       Then the URL "/meshes/default/dataplanes+insights" was requested with
         """
```

Comment:
The changes above here look weird in the diff, almost like tests were removed, but if you look at the files we still cover the same thing.